doc/design/clustered-etcd-alert-store/design.md

```diff
@@ -0,0 +1,150 @@
+# Clustered Etcd Alert Store
+
+Type: Design Document (and PR)
+
+Date: 2019-09-20
+
+Author: David C Wang <dcwangmit01@gmail.com>
+
+## Status Quo
+
+The current Alertmanager clustering implementation will allow multiple
+instances of Alertmanager to sync recently-updated notification log entries and
+alert silences via the gossip protocol.  The syncing of alert state is not yet
+implemented.
```

Comment:
> The syncing of alert state is not **yet** implemented.

So far I was not aware of anyone asking for this to be implemented.

Reply:
Noone has, there have been one or two people who looked for a forwarder of some kind.

Reply:
Acknowledged.  I hope my teams aren't the only ones with this need.  Perhaps we are on the cutting edge of integrating with legacy systems at telcos.

Reply:
I was very surprised that the instances did not sync the alerts. We were looking at using alertmanager outside the usual Prometheus and Kubernetes usage, but this prevented it.